packages/sources/fcsapi/src/adapter.ts

```diff
@@ -23,8 +24,9 @@ const customParams = {
   endpoint: false,
 }
 
-export const execute: Execute = async (input) => {
-  const validator = new Validator(input, customParams)
+// TODO: Run tests with valid API Key, current API Key is expired.
```

Comment:
Has this been bubbled up / handed off?
If not I can assist with this process.

Reply:
No, this has not been handed off, I'm not really sure how to proceed with this so could appreciate your help!